actionpack/test/controller/base_test.rb

```diff
@@ -27,6 +27,16 @@ def default_url_options
   end
 end
 
+class DefaultFormatController < ActionController::Base
+  def show
+    render :nothing => true
```

Comment:
Sure! This test file is full of 1.8 syntax. Is it worth me updating it all in a separate commit? Same for route_set.tb.